install.sh

```diff
@@ -187,14 +186,27 @@ check_deps() {
 node(){
   # clean up 
   rm -rf node 
+  rm -f SHASUMS256.txt.asc
   rm -rf node-$NODE_VERSION*
-  
+
   echo :Installing Node $NODE_VERSION
-  
+
+  gpg --list-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D 2>&1 >/dev/null || gpg --keyserver pool.sks-keyservers.net --recv-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D
```

Comment:
Why not detect if GPG is installed and use it if it is?

This should at LEAST be done over https if GPG is missing.